deepview_profile/analysis/runner.py

```diff
@@ -4,32 +4,41 @@
 from deepview_profile.analysis.session import AnalysisSession
 from deepview_profile.nvml import NVML
 from deepview_profile.utils import release_memory
+import weakref
 
-
-def analyze_project(project_root, entry_point, nvml):
+def analyze_project(project_root, entry_point, nvml, ddp_request):
```

Comment:
Change the name `ddp_request` to something more descriptive. Say `enable_ddp_analysis`.

Reply:
Done